dashboard-app/src/features/dashboard/components/DataCard/util.ts

```diff
@@ -0,0 +1,20 @@
+
+export const sentenceCreator = (xs: string[]) => {
+  switch (xs.length) {
+    case 0:
+    case 1:
+      return xs;
+    case 2:
+      return [xs[0], 'and', xs[1]].map((x) => x.replace('and', ' and '));
```

Comment:
isn't this the same as `[xs[0], ' and ', xs[1]]`, except with the benefit(?) that any strings in `xs` containing the substring `'and'` wont be affected?